pg_backup_api/pg_backup_api/logic/utility_controller.py

```diff
@@ -31,9 +33,19 @@
 from pg_backup_api.run import app
 from pg_backup_api.server_operation import ServerOperation, ServerOperationConfigError
 
+if TYPE_CHECKING:  # pramga: no cover
```

Comment:
Conditionally importing `Request` and `Response` causes the following failure when `TYPE_CHECKING` is `False`:

```
$ ~/.local/bin/pg-backup-api  serve
Traceback (most recent call last):
  File "/var/lib/barman/.local/bin/pg-backup-api", line 11, in <module>
    load_entry_point('pg-backup-api', 'console_scripts', 'pg-backup-api')()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 476, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2700, in load_entry_point
    return ep.load()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2318, in load
    return self.resolve()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2324, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/pg-backup-api/pg_backup_api/pg_backup_api/__init__.py", line 20, in <module>
    import pg_backup_api.logic.utility_controller
  File "/pg-backup-api/pg_backup_api/pg_backup_api/logic/utility_controller.py", line 41, in <module>
    def diagnose() -> Response:
NameError: name 'Response' is not defined
```

Reply:
Missing quoting here too.